### PR TITLE
Implemented new human-readable timestamp field in flow history

### PR DESCRIPTION
**Pull request by nrydanov**
Closes: #4769

### CODE REVIEW

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    public static long REVERSE_METER_ID = 15;
+    public static String REVERSE_PATH = "16";
+    public static String REVERSE_STATUS = "17";
+    public static int SOURCE_INNER_VLAN = 18;
+    public static int SOURCE_PORT = 19;
+    public static String SOURCE_SWITCH = "20";
+    public static int SOURCE_VLAN = 21;
+    public static String TYPE = "22";
+
+    public static FlowEventAction action = new FlowEventAction();
```
Comment (niksv) [inline src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java:74]: i think it's better to put `= new FlowEventAction();` into `initializeData`
As you did with `event`

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        FlowHistoryEntry entry = INSTANCE.map(event, flowHistoryPayloads, flowDumpPayloads);
+
+        assertEquals(entry.getAction(), action.getAction());
+        assertEquals(entry.getDetails(), action.getDetails());
+        assertEquals(entry.getTimestamp(), event.getTimestamp().getEpochSecond());
+        assertEquals(entry.getFlowId(), event.getFlowId());
+        assertEquals(entry.getActor(), event.getActor());
+        assertEquals(entry.getDumps(), flowDumpPayloads);
+        assertEquals(entry.getPayload(), flowHistoryPayloads);
+        assertEquals(entry.getTimestampIso(), event.getTimestamp().atOffset(ZoneOffset.UTC).toString());
```
Comment (niksv) [inline src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java:154]: Also could you pleas add one mere assert.Just to be sure that time is human readable 
```
assertEquals(entry.getTimestampIso(), "2022-04-21T06:41:41.292Z");
```

**Review by dpoltavets — APPROVED** [review]

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java (hunk 1 above)
Comment (nrydanov) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java]: Agree with you, changed

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java (hunk 2 above)
Comment (nrydanov) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java:155]: Okay, done

**Review by dpoltavets — APPROVED** [review]

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+
+public class HistoryMapperTest {
+    public static final HistoryMapper INSTANCE = Mappers.getMapper(HistoryMapper.class);
```
Comment (sergii-iakovenko) [inline src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java:38]: Why not use HistoryMapper.INSTANCE instead?

**Review by sergii-iakovenko — CHANGES_REQUESTED** [review]

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java (hunk 3 above)
Comment (nrydanov) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java]: Yes, it would be better, thanks.

**Review by sergii-iakovenko — APPROVED** [review]

**Pull request merged**